linked_list.rb

```diff
@@ -42,17 +42,41 @@ def search(value)
   # method to return the max value in the linked list
   # returns the data value and not the node
   def find_max
-    puts "Not implemented"
+    current = @head
+    max = current.data

```

Comment:
Check to ensure that head is not nil before accessing data. In other words, update your method to work for empty linked list scenario.